lib/Conversion/TorchOnnxToTorch/DefaultDomainQtoZ.cpp

```diff
@@ -2927,6 +2929,8 @@ void mlir::torch::onnx_c::populateDefaultDomainQtoZ(
           // apparently asymmetric
           if (coordTfMode != "asymmetric" && coordTfMode != "align_corners")
             modeStr = (modeStr + "_") + coordTfMode;
+          if (nearest_mode != "floor" && nearest_mode != "")
```

Comment:
If nearest_mode is empty, I think that means it's the ONNX default (i.e. [round_prefer_floor](https://onnx.ai/onnx/operators/onnx__Resize.html)). But if the nearest_mode of the `torch.aten.__interpolate` op is empty, TorchToLinalg interprets it as floor. Should we change
`binder.customOpNameStringAttr(nearest_mode, "nearest_mode", "")` to `binder.customOpNameStringAttr(nearest_mode, "nearest_mode", "round_prefer_floor")?

Reply:
Thanks for the catch @mgehre-amd !